docs/access-management/sso-providers/azure-entra-id.md

```diff
@@ -1,21 +1,24 @@
 ---
-sidebar_label: Azure Entra ID
```

Comment:
This was on purpose Azure Entra ID, since we explicitaly don't support on prem Microsoft Entra ID but only the Azure version of it. Could you clarify why this was changed to Microsoft Entra ID here?